File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int leftSum = 0;
        int highestAlt = 0;
        
        for(int i=0; i<gain.size(); i++) {
            leftSum += gain[i];
            highestAlt = max(highestAlt, leftSum);
        }
        
        return highestAlt;
    }
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.js
/**
 * @param {number[]} gain
 * @return {number}
 */
var largestAltitude = function(gain) {
    var leftSum = 0;
    var higheshAlt = 0;
    
    for(let i=0; i<gain.length; i++) {
        leftSum += gain[i];
        higheshAlt = Math.max(higheshAlt, leftSum);
    }
    return higheshAlt;
};